packages/web-twig/src/Resources/components/Icon/Icon.twig

```diff
@@ -1,6 +1,6 @@
 {# API #}
 {%- set props = props | default([]) -%}
-{%- set _ariaHidden = props.ariaHidden | default(true) | boolprop -%}
```

Comment:
I am a bit surprised, this is the only place, where we have a true | boolprop :) but it actually is! 

The only thing I am concerned about is, that I didn't find any documentation for this and how/when should you use it.

Reply:
🤷‍♂️

Reply:
So if are already asking me 🙂

- I'd consider the same syntax for all boolean props. When looking into the code, it will be clear to everyone (including future ourselves) that a boolean must be written using `??`. No more thinking about "was it for the `true` value, or for `false`…?".
- There is already `CONTRIBUTING.md` in `web-twig`. I'd like to see this pointed out there.

Reply:
For your first point, there will be a technical panel held for this https://jira.lmc.cz/browse/DS-544 :-)

For the second point, I will add it.

Reply:
Note added.